Source/Moya.swift

```diff
@@ -133,11 +137,16 @@ public class MoyaProvider<Target: MoyaTarget> {
                 cancellableToken.innerCancellable = self.stubRequest(target, request: request, completion: completion, endpoint: endpoint, stubBehavior: stubBehavior)
             }
         }
-
+        
         requestClosure(endpoint, performNetworking)
-
+        
         return cancellableToken
     }
+
+    /// Convenience request-making method that takes no extra endpoint parameters
+    public func request(target: Target, completion: Moya.Completion) -> Cancellable {
```

Comment:
Yeah, breaking changes are fine with me as long as our next version is a major release. As long as we avoid braking semantic versioning, I'm ok with bumping our versions however makes sense for us.